doc/playbook.rst

```diff
@@ -121,10 +121,20 @@ The bucket ``rubin-pp-dev-users`` holds:
 
 * ``rubin-pp-dev-users/unobserved/`` contains raw files that the upload script(s) can draw from to create incoming raws.
 
-``rubin-pp-dev`` has had notifications configured for it; these publish to a Kafka topic.
+``rubin-pp-dev`` has had notifications configured for it; these publish to the Kafka topic ``prompt-processing-dev``.
+The notifications can be viewed at `Kafdrop <https://k8s.slac.stanford.edu/usdf-prompt-processing-dev/kafdrop>`_.
+.
+
+The buckets ``rubin:rubin-pp`` and ``rubin:rubin-pp-users`` are also for Prompt Processing development and previously used by the testers.
```

Comment:
I think the "Buckets" section has grown long enough that it needs to be broken up into sections. Maybe "Current Buckets", "Legacy Buckets", and "AWS Credentials"?

Reply:
Great idea! The section is reorganized and rewritten in https://github.com/lsst-dm/prompt_processing/pull/120/commits/3d07119c17885cb8213e5bd7e998a99f23df9edf